pkg/certificate/certificate_test.go

```diff
@@ -16,40 +12,11 @@ import (
 
 func TestNewFromPEM(t *testing.T) {
 	assert := tassert.New(t)
-	cn := CommonName("Test CA")
-	rootCertCountry := "US"
-	rootCertLocality := "CA"
-	rootCertOrganization := "Root Cert Organization"
 
+	cn := CommonName("Test CA")
```

Comment:
would pass this into the `CreateValidCertAndKey`  to make it explicit that is the CN for that cert